apps/dapp/src/constants/newenv/prod.ts

```diff
@@ -216,10 +228,9 @@ const SPICE_AUCTION_FACTORY_ON_ETH_MAINNET: ContractConfig<SpiceAuctionFactory>
     contractFactory: SpiceAuctionFactory__factory,
   };
 
-// TODO: Prod address
 const SPICE_AUCTION_ON_ETH_MAINNET: ContractConfig<SpiceAuction> = {
   chainId: ETH_MAINNET.id,
-  address: '0x0000000000000000000000000000000000000000',
+  address: '0xBb860920F493ba86a3122110260fd70980e0B2e5',
```

Comment:
This is the spice auction implementation address. From this, the factory clones actual spice auctions for each spice token (eg. [ENA]/[TGLD]. Please check discord for details.